index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <title>ReReddit Eval</title>
+  </head>
+  <body>
+    <div class="row" style="margin-left: 10px">
+      <div class="col-md-6">
+        <div class="page-header">
+          <h1>ReReddit</h1>
+        </div>
+        <div class="posts">
+          <!-- <div class="post">
```

Comment:
Usually you would never want to commit commented out code - you have version control for this, but since we're noobs in git still, it's fine.